app/models/project.rb

```diff
@@ -522,9 +523,9 @@ def allowed_actions
 
   def self.possible_principles_condition
     condition = if Setting.work_package_group_assignment?
-                  ["(#{Principal.table_name}.type=? OR #{Principal.table_name}.type=?)", 'User', 'Group']
+                  ["(#{Principal.table_name}.type=? OR #{Principal.table_name}.type=? OR #{Principal.table_name}.type=?)", 'User', 'Group', 'PlaceholderUser']
                 else
-                  ["(#{Principal.table_name}.type=?)", 'User']
+                  ["(#{Principal.table_name}.type=? OR #{Principal.table_name}.type=?)", 'User', 'PlaceholderUser']
                 end
 
     condition[0] += " AND (#{User.table_name}.status=? OR #{User.table_name}.status=?) AND roles.assignable = ?"
```

Comment:
If the whole method is refactored into a scope, we can again reuse existing scopes. Additionally, we can fix the bug below, where the registered status is not taken into account.

Reply:
Refectored into a `Member` scope